### DISCUSSION TITLE
Missing getLengthOfNationalDestinationCode for [google-libphonenumber] in [@types/google-libphonenumber]

### DISCUSSION THREAD

**Discussion by StrictLine** [DefinitelyTyped/DefinitelyTyped#66772] [Issues with a @types package]
The javascript code from [ruimarinho/google-libphonenumber](https://github.com/ruimarinho/google-libphonenumber) or by installing the [NPM Package](https://www.npmjs.com/package/google-libphonenumber) works just fine and the `PhoneNumberUtil` exposes the "class method" `getLengthOfNationalDestinationCode`, but the type declaration (`index.d.ts`) misses this information.

Location:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/google-libphonenumber/index.d.ts 
Line 171

**Should I fork and open a PR for that?**

Workaround 1 _(turning off type secutity)_:
```ts
(gPhoneLib as any).getLengthOfNationalDestinationCode(parsedNumber)
```

Workaround 2 _(add declaration by interface)_:
```ts
declare module "google-libphonenumber" {
	export interface PhoneNumberUtil {
		getLengthOfNationalDestinationCode(number: libphonenumber.PhoneNumber): number
	}
}
```

[a comment by an automated account is not shown]

**Comment by martin-badin** [comment]
Hi. PR is ready but I am not sure which package version could I use

https://github.com/DefinitelyTyped/DefinitelyTyped/pull/66791

**Reply by StrictLine** [reply]
Hi Martin,
thanks for your PR! I'm a little bit overwhelmed by this question, as I'm new here and in the NPM repo, some version are tagged, some not. I used the @types/google-libphonenumber@7.4.27 with google-libphonenumber@3.2.33, when I stumbled upon this shortage...

**Reply by martin-badin** [reply]
Please review my PR, Thanks
